rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -153,10 +153,77 @@ OAuth2 Authorization Server
 An OAuth2 Authorization Server allows external clients to request to authenticate and act on behalf of users (or as their own identities). Having
 an OAuth2 Authorization Server enables Flyte administrators control over which apps can be installed and what scopes they are allowed to request or be granted (i.e. what privileges can they assume).
 
+BuiltIn Authorization Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 Flyte comes with a built-in authorization server that can be statically configured with a set of clients to request and act on behalf of the user.
-The default clients are defined `here <https://github.com/flyteorg/flyteadmin/pull/168/files#diff-1267ff8bd9146e1c0ff22a9e9d53cfc56d71c1d47fed9905f95ed4bddf930f8eR74-R100>`__
+The default clients are defined `here <https://github.com/flyteorg/flyteadmin/blob/12d6aa0a419ccec81b4c8289fd172e70a2ded525/auth/config/config.go#L86-L110>`__
 and the corresponding section can be modified through configs.
 
+By default, the BuiltIn Authorization Server ships with 3 statically configured clients:
+
+1. **flyte-cli**: A public OAuth2 client that can be used with flyte-cli command line tool. This client does need a client_secret.
+
+2. **flytectl**: A public OAuth2 client that can be used with flytectl command line tool. This client does need a client_secret.
+
+3. **flytepropeller**: A client-credentials OAuth2 client that can be used with flytepropeller to talk to flyteAdmin to publish execution events.
+   The default secret set to `foobar`. In order for flytepropeller to pickup that secret, you will need to set the secret as follows:
```

Comment:
Should this default value be specified? This PR removes `foobar` as a default.

Reply:
it's still set here: https://github.com/flyteorg/flyteadmin/blob/12d6aa0a419ccec81b4c8289fd172e70a2ded525/auth/config/config.go#L105